Resources/frontend/js/jquery.adyen-confirm-order.js

```diff
@@ -120,11 +120,9 @@
                     break;
                 case 'IdentifyShopper':
                 case 'ChallengeShopper':
-                    me.handleThreeDs(data);
-                    break;
                 case 'Pending':
                 case 'RedirectShopper':
-                    me.handlePaymentDataRedirectShopper(data);
+                    me.handlePaymentDataCreateFromAction(data);
```

Comment:
Hey @cyattilakiss do you think we should add PresentToShopper here as well?

Reply:
Let's move this offline because this might be a bigger topic to discuss and I have a couple of follow up questions as well.

Reply:
We won't add PresentToShopper just yet as it needs to be handled differently, so now it is being handled by the default switch case and `handlePaymentDataError`